hook-worker/src/worker.rs

```diff
@@ -234,6 +235,10 @@ async fn process_webhook_job<W: WebhookJob>(
 
     match send_result {
         Ok(_) => {
+            let insert_to_complete_duration = Utc::now() - webhook_job.job().created_at;
+            metrics::histogram!("webhook_jobs_insert_to_complete_duration_seconds", &labels)
```

Comment:
could we please add the retry count to the metric label, so that we can filter out by `retries=0` for the happy path? Otherwise, the metric might be pretty noisy due to retries, but it's still interesting to graph the p95 by `retries`.